### ISSUE TITLE
Window is undefined when running in node environment

### ISSUE THREAD

**Issue by meltedspark** [meltedspark/jest-marbles#7]
When running jest with `testEnvironment: 'node'` any import from `jest-marbles` causes the following exception:

```js
    ReferenceError: window is not defined

    > 1 | (function webpackUniversalModuleDefinition(root, factory) {
      2 |       if(typeof exports === 'object' && typeof module === 'object')
      3 |               module.exports = factory();
      4 |       else if(typeof define === 'function' && define.amd)

      at Object.<anonymous> (../umd/jest-marbles.js:1:412)
      at Object.<anonymous> (umd-import.spec.js:1:118)
```

**Comment by meltedspark** [comment]
Related bug in webpack: https://github.com/webpack/webpack/issues/6677

**Comment by meltedspark** [comment]
Patched in 1.0.5 bundle published on npm. Fix for the source code awaiting Webpack fix.
